src/VIPServer.java

```diff
@@ -107,24 +107,6 @@ public void main(String[] args) throws HostFailureException, HostNotFoundExcepti
 					// SHOULD BE REPLACED BY
 					// job.carryOn();
 				} else {
-					if (!timer) {
```

Comment:
So to conclude, I will add a new flag on the simulator command line to distinguish the 2 modes (Static VS Dynamic).
 1) In static,  Gate jobs directly enter END after computation and a Timer in VIPServer to start merge( as I have added in GATE_DISCONNECT)
  2) In dynamic, Gate jobs send PROGRESS and I add back the block that i removed in GATE_PROGRESS ?
    Is that correct and you all agree with these modifs?